src/chi/javascript/components/mobile-navigation.js

```diff
@@ -319,7 +319,7 @@ class MobileNav extends Component {
 
   hide() {
     const firstLevelDrawer = this._firstLevelDrawer;
-    const secondLevelDrawer = this._drawers.find(drawer => Util.hasClass(drawer, chi.classes.ACTIVE));
+    const secondLevelDrawer = this._drawers.filter(drawer => Util.hasClass(drawer, chi.classes.ACTIVE))[0];
```

Comment:
This will fail with an error if there is no secondLevelDrawer. If it is mandatory to exist, then this code is right. If not, you'll need to check if the secondLevelDrawer exists before getting the first element with the `[0]`.

Reply:
I'm sorry. I was wrong. It won't happen again 🤴

Reply:
🏃